### PR TITLE
Emaildevs button!

### PR DESCRIPTION
**Pull request by petegriffith**
This is a simple button located in the profile page that allows a user to email fruityraptors@gmail.com.

Because we're still in development, we really need a way to get feedback from users. This should take care of that!

### CODE REVIEW

**Review by Evomatic — APPROVED** [review]
Great idea, interesting implementation

File: src/views/Profile.vue [hunk 1]
```diff
@@ -1,90 +1,104 @@
...
 
       <v-snackbar v-model="snackbar" timeout="2000">
-      Copied to Clipboard!
+        Copied to Clipboard!
       </v-snackbar>
 
-      
+      <v-avatar size="83" class="mb-3">
+        <Avatar
+          v-if="!this.$store.state.user.photo_url"
```
Comment (KuroKen91) [inline src/views/Profile.vue:17]: Would it be best to place this store data in data or computed? Email and username seemed to be getting fetched so perhaps we could streamline the process to keep the code up here short/consistent?

**Review by KuroKen91 — APPROVED** [review]
LGTM!

File: src/views/Profile.vue (hunk 1 above)
Comment (petegriffith) [reply src/views/Profile.vue:17]: Since this isn't an issue with the emaildevs button, let's take it up elsewhere?

File: src/views/Profile.vue [hunk 2]
```diff
@@ -1,90 +1,104 @@
...
+    <v-card
+      class="mt-2 mb-2 d-flex justify-center orange lighten-4"
+      elevation="0"
+    >
+      <a href="mailto:fruityraptors@gmail.com" target="_blank" class="email-button"
+        ><v-btn color="green lighten-2 black--text mb-5">
+          Email the dev team
+        </v-btn></a
+      >
+    </v-card>
```
Comment (jaysutham) [inline src/views/Profile.vue:94]: Super nitpicky... but

1. proper indentation
2. maybe added `rounded` to v-btn so it matches the rest

**Review by jaysutham — CHANGES_REQUESTED** [review]

File: src/views/Profile.vue (hunk 2 above)
Comment (petegriffith) [reply src/views/Profile.vue:94]: will do this now...

**Pull request merged**